src/atoms/Banner.tsx

```diff
@@ -0,0 +1,83 @@
+import type { FunctionComponent, PropsWithChildren } from 'react';
+import { useState } from 'react';
+
+import { Body, Box, Flex, Image } from '.';
+import error from '../assets/icons/alert-red.svg';
+import caret from '../assets/icons/caret.svg';
+import success from '../assets/icons/circle-checkmark.svg';
+import warning from '../assets/icons/circle-warning.svg';
+import info from '../assets/icons/info.svg';
+import action from '../assets/icons/queue-waiting.svg';
+import { Badge } from './Badge';
+
+export type BannerType = 'success' | 'info' | 'action' | 'warning' | 'error' | 'clear';
+
+export interface BannerProps {
+  type: BannerType;
+  label: string;
+  badge?: string;
+  extended?: boolean;
+}
+
+const icons: { [key in BannerType]: string } = {
+  success,
+  info,
+  action,
+  warning,
+  error,
+  clear: action
+};
+
+export const Banner: FunctionComponent<BannerProps> = ({
+  type,
+  label,
+  badge,
+  extended,
+  children
+}: PropsWithChildren<BannerProps>) => {
+  const [isExtended, setExtended] = useState(extended);
+
+  const handleToggle = () => setExtended((value) => !value);
+
+  return (
+    <Box mb="2" variant={`banner.${type}`} sx={{ borderRadius: 'banner' }}>
+      <Flex
+        variant="horizontal-start"
+        p="2"
+        px={type === 'clear' && '0'}
+        justifyContent="space-between"
+        onClick={children && handleToggle}
+        sx={{ cursor: children && 'pointer' }}
+        data-testid="banner-toggle">
+        <Flex variant="horizontal-start">
+          <Image src={icons[type]} alt="type" width="20px" minWidth="20px" height="20px" mr="2" />
+          <Body
+            color="inherit"
+            fontSize="12px"
```

Comment:
If possible use one of the font sizes defined in the theme. Not sure if they match up

Reply:
Done